src/main/java/reposense/util/FileUtil.java

```diff
@@ -199,7 +204,8 @@ public static void unzip(InputStream is, Path outputPath) throws IOException {
     }
 
     /**
-     * Copies the template files from {@code sourcePath} to the {@code outputPath}.
+     * Copies the template files from {@code is} to the {@code outputPath}.
```

Comment:
```suggestion
     * Copies the template files from the {@code is} to the {@code outputPath}.
```
Standardize with line 176.